src/libs/actions/App.js

```diff
@@ -79,14 +88,46 @@ AppState.addEventListener('change', (nextAppState) => {
     appState = nextAppState;
 });
 
-function triggerUpdateAvailable() {
-    Onyx.set(ONYXKEYS.UPDATE_AVAILABLE, true);
+/**
+ * Fetches data needed for app initialization
+ *
+ * @param {Boolean} shouldSyncPolicyList Should be false if the initial policy needs to be created. Otherwise, should be true.
+ * @returns {Promise}
+ */
+function getAppData(shouldSyncPolicyList = true) {
+    NameValuePair.get(CONST.NVP.PRIORITY_MODE, ONYXKEYS.NVP_PRIORITY_MODE, 'default');
+    NameValuePair.get(CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER, ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER, true);
+    getLocale();
+    User.getUserDetails();
+    User.getBetas();
+    User.getDomainInfo();
+    PersonalDetails.fetchLocalCurrency();
+    GeoLocation.fetchCountryCodeByRequestIP();
+    BankAccounts.fetchFreePlanVerifiedBankAccount();
+    BankAccounts.fetchUserWallet();
+
+    if (!UnreadIndicatorUpdater.isListeningForReportChanges()) {
+        UnreadIndicatorUpdater.listenForReportChanges();
+    }
```

Comment:
Sorry, to clarify, I think this was in the correct spot before and does not belong in this function.

Reply:
🤦  That makes sense! Thanks!